### PR TITLE
Provide better Fiji interoperability

### PR DESCRIPTION
**Pull request by timholy**
This fixes issues in #10 and adds much better interoperability with NRRD files written by Fiji.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -106,7 +106,9 @@ include("unu-make.jl")
     end
 
     gc()  # to close any mmapped files
-    rm(workdir, recursive=true)
+    try
+        rm(workdir, recursive=true)
```
Comment (timholy) [inline test/runtests.jl]: ...tried my best :wink:

**Comment by timholy** [comment]
OSX is stalled as usual, but I don't foresee any problems.

File: test/runtests.jl [hunk 2]
```diff
@@ -87,8 +87,28 @@ include("unu-make.jl")
...
+            fn = joinpath(dirname(@__FILE__), "io", name)
+            v = load(fn)
+            @test size(v) == (50,40,30)
+            @test pixelspacing(v) == ps
+        end
+    end
+
     gc()  # to close any mmapped files
-    rm(workdir, recursive=true)
+    try
```
Comment (tkelman) [inline test/runtests.jl:109]: that's not much of a fix, do you still have a handle or reference to the mmap'ed array open? setting it to nothing and a few extra gc calls, and/or manually finalizing it should help

Comment (timholy) [reply test/runtests.jl:109]: Well, the variable is out of scope and there's a `gc` call directly above this line.

Comment (tkelman) [reply test/runtests.jl:109]: sometimes it requires multiple calls to trigger a full sweep or something

Comment (timholy) [reply test/runtests.jl:109]: Looks like `gc(true)` will force a full sweep. I'll add that in my next commit, but meanwhile this is valid code, and in particular you don't want the tests failing simply for failing to clean up. In other words, I plan to leave this `try ... end` block in place after I change it to `gc(true)`.

**Pull request merged**